src/utils/immutable.js

```diff
@@ -0,0 +1,56 @@
+/** General helpers to augment Immutable.js. */
```

Comment:
Just to point it out in case it's helpful (I don't have a strong preference): would you prefer `src/immutableUtils.js` as a name for this? See discussion: https://github.com/zulip/zulip-mobile/pull/4683#discussion_r665879420

Reply:
Sure, thanks for the reminder of that thought.